README.md

```diff
@@ -35,7 +35,7 @@ The code has been developed and tested on an Ubuntu 14.04 system with ROS Indigo
 The code is built in a standard ROS fashion. In your ROS workspace root execute:
 
  ```
-  catkin_make --pkg example_ros_cpp
+  catkin build example_ros_cpp
```

Comment:
This only works on ROS versions newer than Jade. Should we keep the instructions for Indigo as well ?